### ISSUE TITLE
Bugs in font name handling outside libass

### ISSUE THREAD

**Issue by LanceNorskog** [libass/libass#306]
This type of error report occurs often in Handbrake. 

`[17:37:21] [ass] fontconfig: Selected font is not the requested one: 'KozMinPro-Strippedv2-Regular' != 'Kozuka Mincho Pro Strippedv2 R'`

The attachment inside the MKV file for this font is named: `KozMinPro-Strippedv2-Regular.ttf`
The ASS code in the video uses this as the font name: `Kozuka Mincho Pro Strippedv2 R`

Obviously, it's not a libass bug. Presumably the subtitle editor (aegisub) is mangling font names in two different ways. But, is it possible to add a more forgiving font-finder?

**Comment by TheOneric** [comment]
@LanceNorskog   Can you provide more info? Does this still persist?
Searching for the exact filename `KozMinPro-Strippedv2-Regular.ttf` yielded a few sites where this font can be downloaded, though confusingly some actually contained a different font. Using one one that actually contained the advertised font, I'm unable to reproduce this. I just get the below, with no further font-related messages:
```
libass: fontselect: (Kozuka Mincho Pro Strippedv2 R, 400, 0) -> ~/.fonts/KOZMINPRO-STRIPPEDV2-REGULAR.TTF, 0, KozMinPro-Strippedv2-Regular
libass: fontselect: (KozMinPro-Strippedv2-Regular, 400, 0) -> ~/.fonts/KOZMINPRO-STRIPPEDV2-REGULAR.TTF, 0, KozMinPro-Strippedv2-Regular
```

For reference here's what fontforge shows:
```
Fontname:         KozMinPro-Strippedv2-Regular
Familyname:       Kozuka Mincho Pro Strippedv2 R
Name for Humans:  KozMinPro-Strippedv2-Regular
Weight:           Book
```

**Comment by TheOneric** [comment]
Actually the message-string `Selected font is not the requested one` was already replaced by 28908369203be43c32bd1ac499fa3dc825b78c96 included in release 0.10.2 back in 2013 — **5 years** and several releases before this issue was opened.
 The new error message should be more descriptive, but I'm not getting any of those either.

**Comment by LanceNorskog** [comment]
Hi- I have stopped using HB to burn styled subtitles into anime, since I upgraded my video player to something that (probably) has a modern libass. Also, I'm was using HB on an old Linux Mint (#13) which stopped taking updates for various things awhile back.

Apologies for wasting your time. I wrote the 'sox' program 30 years ago, and let someone else take it over. I have the greatest respect for people who keep working on old projects :) Closing.
